assets/js/base/components/price-slider/submit-button.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import PropTypes from 'prop-types';
+
+const SubmitButton = ( { disabled, onClick } ) => {
+	return (
+		<button
+			type="submit"
+			className="wc-block-price-filter__button wc-block-form-button"
+			disabled={ disabled }
+			onClick={ onClick }
+		>
+			{ __( 'Go', 'woo-gutenberg-products-block' ) }
```

Comment:
Due to the ambiguousness of this label, should there be a translator comment here?